### DISCUSSION TITLE
v1.1.0-preview1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2331] [Announcements]
### 1.1 Preview 1 Release of Windows App SDK 

We've just released our first preview of the 1.1 release of the Windows App SDK! This preview includes several new capabilities, including:
- Support for running applications with elevated privileges
- Self-contained app deployment
- Support for multi-window applications (also available in version 1.0.1 and newer)
- Push and app notifications
- Enabling developers to add, remove, and modify environment variables without having to directly use the registry API.

**Known issue for WinUI 3 applications:** Users are unable to drop an element when drag and drop is enabled.

To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.1/1.1.0-preview1/release-notes). 

### What's next 
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://aka.ms/winappsdkroadmap), which gets updated regularly.  

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.0-preview1'>v1.1.0-preview1</a>.</em>

**Comment by riverar** [comment]
Congrats team on shipping 1.1-preview1. Self-contained app deployment is huge, taking it for a spin now.

**Comment by shelllet** [comment]
great!

**Comment by shelllet** [comment]
why only windows 11 ?
![image](https://user-images.githubusercontent.com/16729017/160744266-cacc22c8-a3a0-4364-87d1-8857e2746bd7.png)

**Reply by riverar** [reply]
Believe the issue was fundamentally an OS issue that didn't get fixed until Windows 11. Looping in @DrusTheAxe who knows more.

**Reply by shelllet** [reply]
if this, i had to stop playing winui3 and pick up qt. i can give up win7, but  can't give up win10 at the moment，

**Reply by shelllet** [reply]
I don't know why the dev team gave up on win10?
![image](https://user-images.githubusercontent.com/16729017/160769681-f62cf6c5-4f29-42e4-b5ea-0068e45ec969.png)

**Reply by BenJKuhn** [reply]
We still love Win10. ;) 

We’re evaluating a change to supported versions of Windows 10 that will enable elevation there as well. We’ll let you know more once plans are solidified. 

- Ben

**Reply by Pinox** [reply]
yup, essentially we have to wait at least 1.5 years before this becomes a viable feature to roll out and that's not great.  On the bright side Windows 11 adoption on current trend is about 6% a month and that is great.

**Comment by ghost1372** [comment]
where is mica? Still not available?

**Comment by ThierryFourniol** [comment]
Thank you so much for the v1.1 Preview 1 as the Self-contained app deployment is an amazing capability... except:
Known issue for WinUI 3 applications: Users are unable to drop an element when drag and drop is enabled.
This is tragic as my drag and drop desktop app is not fit for purpose anymore!
I confirm Dropping an element is crashing the app indeed!
So please can you fix it ASAP? Or is there any quick workaround? Please 'save my life'. Thank you!

**Reply by BenJKuhn** [reply]
Is this the issue you're referring to? Let me see if I can get an update on status.

https://github.com/microsoft/microsoft-ui-xaml/issues/4574

Thanks,

Ben

**Reply by ThierryFourniol** [reply]
Yes, exactly.
For me, it is in case of dropping a ListView item to another ListView item, which was working perfectly with v1.0, before switching to the v1.1 Preview 1.

Thanks, Ben.

Thierry

**Reply by BenJKuhn** [reply]
(moving reply to the right place)

That issue dates back to Reunion 0.5, which is super-early. If you're seeing a new crash / regression upgrading from 1.0 to 1.1-preivew1, even if it seems similar to the existing issue, can you file it as a new issue? I'd like to make sure we track the fact that something "went backwards" on upgrade, and make sure we're not making the problem worse.

(update)
I also confirmed that the original issue is being tracked and is still in the backlog, but does not have an ETA yet.

Ben
(FYI @MikeHillberg)

**Comment by ThierryFourniol** [comment]
Self-contained works well with the Preview 1, Great and many thanks for that,
but is there a way to get a single .exe file instead of about 344 files next to the .exe file?
Because setting PublishSingleFile to True does not work for me, sadly.
Has anyone managed to achieve this? Thanks!

**Reply by ghost1372** [reply]
single exe is not work
https://github.com/microsoft/WindowsAppSDK/issues/2353

**Reply by ThierryFourniol** [reply]
Thank you for letting me know.
My case is similar to yours: unpackaged app with InfoBar.
So what is the workaround:
should we replace all our InfoBar by another control such as ContentDialog? Maybe...
